requirements.txt

```diff
@@ -1,4 +1,5 @@
 # Extra packages required to install and use the tools
+black==19.3b0
```

Comment:
This change (and the `setup.py` changes) still haven't been merged, I'm happy to quickly open another PR to do this